hack/make/build-rpm

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+
+# subshell so that we can export PATH without breaking other things
+(
+	source "$(dirname "$BASH_SOURCE")/.integration-daemon-start"
+
+	# TODO consider using frozen images for the dockercore/builder-rpm tags
+		
+	rpmName=docker-engine
+	rpmVersion="${VERSION%%-*}"
+	rpmRelease=1
+
+	# rpmRelease versioning is as follows
+	# Docker 1.7.0:  version=1.7.0, release=1
+	# Docker 1.7.0-rc1: version=1.7.0, release=0.1.rc1
+	# Docker 1.7.0-dev nightly: version=1.7.0, release=0.0.YYYYMMDD.HHMMSS.gitHASH
+	
+	# if we have a "-rc*" suffix, set appropriate release
+	if [[ "$VERSION" == *-rc* ]]; then
+		rcVersion=${VERSION#*=rc}
+		rpmRelease="0.${rcVersion}.rc${rcVersion}"
```

Comment:
Oh clever.  I was thinking `0.1.something` meant RC just to make it +1 over nightlies, since `rc1` and `rc2` will automatically sort appropriately.  My only concern here is if we ever think we might have `rc0`, which I think is possible.

@vbatts does `rpm` have something like `dpkg --compare-versions` for easily checking how these things sort?  ie, `dpkg --compare-versions 1.0.0 '>' 1.0.0~rc1 && echo yes || echo no` prints `yes`